### PR TITLE
Building with Backgrounds and Gradients Project: apple.com web page clone

### PR DESCRIPTION
**Pull request by Azeem838**
This pull request includes the following:

1. Navbar with gradient apple.com colors
2. Grid and FlexBox layouts
3. Background image

This is a request for a review of our apple.com webpage clone. Kindly find our commits and code below.

Thank you.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+}
+
+.line-3 svg {
+  padding-left: 10px;
+}
+
+.nav div svg {
+  /* position: absolute;
+  top: 2px;
+  left: -12px; */
```
Comment (christianotieno) [inline style.css]: # [REQUIRED] - REMOVE - Commented Out Code
- ***Commented Out Code*** need not be part of your final codes. You can use git history to track for changes and review previous version of your code. Kindly remove all the [commented code](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) in your files including this one. 👆

**Review by christianotieno — CHANGES_REQUESTED** [review]
# **CHANGES REQUIRED** :warning:
Hey @Azeem838  :wave:
-
You have done a splendid job! :tada: I love how clean and modular your code is. Keep it up.
We, however, do have some few changes to carry out before we proceed (see review comment).
##

Please makes changes and submit another review request.
- If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email. 👀 
##
- When you have made the specified changes, please submit another code review. Thank you!
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)
---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Review by christianotieno — CHANGES_REQUESTED** [review]
More changes below.

File: README.md [hunk 2]
```diff
@@ -1,57 +1,29 @@
-# Project Name
+# Apple.com Webpage Clone
 
-> One paragraph statement about the project.
+> This project was to clone the Apple.com webpage using backgrounds and gradient colors.
 
-![screenshot](./app_screenshot.png)
+![screenshot](/images/apple-webpage.png)
```
Comment (christianotieno) [inline README.md]: # [REQUIRED] - FIX - Link to a screenshot.

Change your link as shown below to allow anyone viewing your PR see the screenshot.
```suggestion
![screenshot](./images/apple-webpage.png)
```

This is what I see: 
screenshot

File: README.md [hunk 3]
```diff
@@ -1,57 +1,29 @@
...
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+[Live Demo Link](https://azeem838.github.io/apple.com-webpage/)
```
Comment (christianotieno) [inline README.md]: # [REQUIRED] - FIX - Live link to your repo.
The link is not working. Kindly fix that,

**Review by christianotieno — CHANGES_REQUESTED** [review]

**Comment by SofiBretz** [comment]
Well done @Azeem838! 👏

Project Approved!
You did all the changes required previously. 
Good Luck with your next project. Happy coding!

You can merge 👍
SofiBretz.

**Pull request merged**